packages/AutoTDD-GUI.package/ATDDBrowser.class/class/watcherFunctionalityDictionary.st

```diff
@@ -3,12 +3,12 @@ watcherFunctionalityDictionary
 	
 	^ OrderedDictionary
 		newFrom: {
-			'Add watcher' -> #addWatcher.
-			'Pause' -> #pauseSelectedWatchers.
+			'Add watcher' -> #createWatcher.
+			'Pause' -> #pauseWatchers.
 			'Resume' -> #resumeSelectedWatchers.
 			'Run' -> #resumeSelectedWatchers.
-      		'Edit' -> #editWatcher.
-			'Info' -> #openInfoForSelectedWatchers.
-			'Results' -> #openResultsForSelectedWatchers.
+      		'Edit' -> #editWatchers.
+			'Info' -> #infoWatchers.
+			'Results' -> #resultsWatchers.
 			'Remove' -> #removeSelectedWatchers.
```

Comment:
We should probably be consistent with periods on the last line of ordered dictionaries. For instance, this function does have a period on the last line, however the method `testingEntityResultsDictionary` from the same class does not. Perhaps there is something I'm not seeing?